integration-test/experimental-react/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@integration-test/experimental-react",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "prepare": "rm -rf node_modules/@apollo/client-react-streaming; mkdir node_modules/@apollo/client-react-streaming && cp -r ../../packages/client-react-streaming/{package.json,dist} node_modules/@apollo/client-react-streaming",
```

Comment:
This is necessary so it doesn't pick up the `react` folder from `packages/client-react-streaming/node_modules` and mixes two react versions.